internal/http/retry.go

```diff
@@ -65,5 +70,17 @@ func RetryPolicy(ctx context.Context, resp *http.Response, err error) (bool, err
 		return true, nil
 	}
 
+	// Check for json response and description for retries.
+	// i.e.: when receiving TOO_MANY_REQUESTS it is a HTTP 200 code with an
+	// error status that needed to be retried.
+	json, jErr := io.ReadAll(resp.Body)
+	resp.Body = io.NopCloser(bytes.NewBuffer(json))
+	if jErr == nil {
+		errorClass := fastjson.GetBytes(json, "errors", "0", "extensions", "errorClass")
+		if errorClass != nil && bytes.Compare(errorClass, []byte("TOO_MANY_REQUESTS")) == 0 {
```

Comment:
🤔 Thoughts on using `bytes.Equal()` in the check here?

Reply:
As the error was changed to a `const` on the package, it looked cleaner to use `GetString` and not bytes anymore.